DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/GroupDelegates/_AddGroupDelegateCard.cshtml

```diff
@@ -0,0 +1,90 @@
+
+  <details class="nhsuk-details nhsuk-expander">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.DelegateInfo.Id-name">
+        <span class="searchable-element-title" name="name">@Model.DelegateInfo.Name</span>@emailString
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+
+      <partial name="SearchablePage/_FilterableTags" model="@Model.Tags" />
+
+      <dl class="nhsuk-summary-list details-list-with-button">
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">
+          Name
+        </dt>
+        <partial name="_SummaryFieldValue" model="@Model.DelegateInfo.Name" />
+    </div>
+
+    <div class="nhsuk-summary-list__row details-list-with-button__row">
+      <dt class="nhsuk-summary-list__key">
+        Email
+      </dt>
+      <partial name="_SummaryFieldValue" model="@Model.DelegateInfo.Email" />
+    </div>
+
+    <div class="nhsuk-summary-list__row details-list-with-button__row">
+      <dt class="nhsuk-summary-list__key">
+        ID
+      </dt>
+      <partial name="_SummaryFieldValue" model="@Model.DelegateInfo.CandidateNumber" />
+    </div>
+
+    <div class="nhsuk-summary-list__row details-list-with-button__row">
+      <dt class="nhsuk-summary-list__key">
+        Registration date
+      </dt>
+      <partial name="_SummaryFieldValue" model="@Model.DelegateInfo.RegistrationDate" />
+      <div hidden name="registration-date">@Model.DelegateInfo.RegistrationDate</div>
+    </div>
+
+    <div class="nhsuk-summary-list__row details-list-with-button__row">
+      <dt class="nhsuk-summary-list__key">
+        Job group
+      </dt>
+      <partial name="_SummaryFieldValue" model="@Model.DelegateInfo.JobGroup" />
+      <div hidden data-filter-value="@Model.JobGroupFilter"></div>
+    </div>
+
+    @foreach (CustomFieldViewModel customField in Model.DelegateInfo.CustomFields)
+    {
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">@customField.CustomPrompt</dt>
+        <partial name="_SummaryFieldValue" model="@customField.Answer" />
+        @if (Model.CustomPromptFilters.ContainsKey(customField.CustomFieldId))
+        {
+          <div hidden data-filter-value="@Model.CustomPromptFilters[customField.CustomFieldId]"></div>
+        }
+      </div>
+    }
+    </dl>
+
+
+</div>
+  </details>
+
+<div class="flex-row nhsuk-u-margin-left-4" aria-hidden="false">
+  <form method="post" aria-hidden="false">
+    <input type="hidden" name="delegateId" value="@Model.DelegateInfo.Id" />
+    <button class="nhsuk-button nhsuk-u-margin-bottom-2 nhsuk-u-margin-right-2"
+            asp-controller="GroupDelegates"
+            asp-action="AddDelegate"
+            asp-route-groupId="@ViewData["groupId"]"
+            asp-route-delegateId="@Model.DelegateInfo.Id">
+      Add to group
+    </button>
+  </form>
+</div>

```

Comment:
Some odd indentation going on down here.

Reply:
Formatter has fixed it for me.